daq/panda/testing/daqTestScript.py

```diff
@@ -45,9 +45,13 @@
 #Account for 1600 ms delay in Arduino connection
 sleep(1.6)
 
-numValues = 22;
 n = 0
 while(n<50):
```

Comment:
What's the purpose of `n`? Do you intend to essentially test the code 50 times? If so, maybe `num_runs` would be better?